src/data/types.ts

```diff
@@ -43,14 +43,6 @@ export type PineconeConfiguration = {
   additionalHeaders?: HTTPHeaders;
 };
 
-/** Configuration for a single Pinecone Index */
```

Comment:
This was kind of a silly thing to implement originally, and I don't think we really need a type to represent this.